app/assets/javascripts/discourse/tests/acceptance/preferences-test.js

```diff
@@ -329,26 +298,6 @@ acceptance(
   }
 );
 
-acceptance("User Preferences when badges are disabled", function (needs) {
```

Comment:
The assertion in this tests make no sense as to how it relates to badges being disabled.